docker-compose/docker-compose.yml

```diff
@@ -2,14 +2,14 @@ version: '2'
 
 services:
   dse-1:
-    image: datastax/dse-server:${DSETAG}
+    image: stargateio/dse-next:${DSETAG}
     networks:
       - stargate
     mem_limit: 2G
     environment:
       - MAX_HEAP_SIZE=1536M
-      - CLUSTER_NAME=dse-${DSETAG}-cluster
-      - DC=dc1
+      - CASSANDRA_CLUSTER_NAME=dse-${DSETAG}-cluster
```

Comment:
Apparently OSS Cassandra uses "CASSANDRA_CLUSTERNAME" but DSE "CLUSTER_NAME". Ayayay.

We may need to change this again if and when we switch to upcoming DSE 7.0.